pyproject.toml

```diff
@@ -108,7 +109,7 @@ select = [
   "I001", # isort
   "I002", # isort
 ]
-ignore = ["RUF012"]
+ignore = ["RUF012", "RUF013", "PLR0913", "PLR0915", "PLR0912", "PLR0911", "E501"]
```

Comment:
we should avoid ignoring ruff validations, otherwise we will deacrese quality of the code.